Cargo.toml

```diff
@@ -129,9 +129,9 @@ static-files = "0.2.3"
 strum = "0.26.3"
 temp-env = "0.3"
 tempfile = "3"
-test-context = "0.3"
+test-context = "0.4"
 test-log = "0.2.16"
-thiserror = "1.0.58"
+thiserror = "1.0.69"
```

Comment:
Might be worth switching to v2 of that.

Reply:
agreed :+1:  

* I mainly focused on utoipa for this pr + other easy/safe to update deps
* https://github.com/dtolnay/thiserror [has 1x and 2x](https://github.com/dtolnay/thiserror/releases?page=2) with no central change_log

i'll take a look on that